src/libraries/targetframework.props

```diff
@@ -76,7 +76,7 @@
   </Choose>
   
   <PropertyGroup>
-    <TargetsNetCoreApp Condition="$(TargetFramework.StartsWith('netcoreapp'))">true</TargetsNetCoreApp>
+    <TargetsNetCoreApp Condition="$(TargetFramework.StartsWith('netcoreapp')) or $(TargetFramework.StartsWith('net5.'))">true</TargetsNetCoreApp>
```

Comment:
Would it be better to change this to, or `(!$(TargetFramework.StartsWith('net4')) and !$(TargetFramework.StartsWith(netstandard)))` that would make it resilient to future TFMs, `net6`, `net7` etc. Also, we could move this property declaration after the `TargetsNetfx` and `TargetsNetstandard` and use those properties to set this variable as well.

Reply:
This is just a short term workaround. @Anipik is working on changing/removing these properties as they read from `TargetFramework` before the project is evaluated. https://github.com/dotnet/runtime/issues/34525

Reply:
but I guess we will be smarter as how we declare them, right? Or is the plan to eliminate them at all?

Reply:
So they'll only be set after properties are evaluated and we could either keep them and define from TargetFrameworkIdentifier, or just eliminate in favor of TargetFrameworkIdentifier.

Reply:
Ok, just something to consider here if we keep them so that we don't have to keep updating the `TargetsNetCoreApp` property every change of TFM.

Reply:
right, I guess we will just remove them...

Reply:
The mentioned work needs to be finished long before we move to net6, so I'll leave it as is for now.